templates/Gemfile.erb

```diff
@@ -7,7 +7,10 @@ end
 
 ruby "<%= Suspenders::RUBY_VERSION %>"
 
+<% unless options[:api] %>
 gem "autoprefixer-rails"
+<% end %>
+
```

Comment:
Maybe cut this empty line, so that on output there isn't an extra line between gems? We seem to do this on Line 27, as well, with the webpack option.